.travis.yml

```diff
@@ -19,8 +19,11 @@ services:
 before_install:
   - mkdir -p $HOME/.hypothesis
   # If necessary, set up an appropriate version of pypy.
-  - if [ ! -z "$PYPY_VERSION" ]; then source setup-pypy-travis.sh; fi
-  - if [ ! -z "$PYPY_VERSION" ]; then python --version 2>&1 | fgrep "PyPy $PYPY_VERSION"; fi
+  - >
+     if [[ -n "$PYPY_VERSION" ]]; then
```

Comment:
Can we be sure that Travis uses `bash` or some other shell that supports `[[`? If not, it's probably safer to stick with `[` instead.